ports/atmel-samd/boards/capablerobot_usbhub/mpconfigboard.mk

```diff
@@ -0,0 +1,18 @@
+LD_FILE = boards/samd51x19-bootloader-external-flash.ld
+USB_VID = 0x04D8
+USB_PID = 0xEDB3
```

Comment:
Are you sure it's ok to use a MicroChip VID?

Reply:
I went through the sublicense process and was approved.  

![Screen Shot 2019-05-14 at 2 56 55 PM](https://user-images.githubusercontent.com/10534/57724540-03d57b00-7659-11e9-8bc1-32ec28a04665.png)

Reply:
Interesting! Thank you for letting me know.